readme.md

```diff
@@ -7,10 +7,12 @@
 - Ansible 2.x (http://docs.ansible.com/ansible/intro_installation.html)
 - Vagrant (https://www.vagrantup.com/docs/installation/)
 - NFS
+- Yarn
 
 ## Getting started
 1. Create a mapping in your hosts file from phpsw.local to 192.168.42.10.
-2. ```vagrant up```
+1. Run `yarn install` and either `yarn run encore dev` or `yarn run encore dev --watch` to generate the front-end assets.
+1. ```vagrant up```
```

Comment:
I think the order of this is wrong. We should be running `yarn` in the VM not the host.  

Should it not be something like:

- `vagrant up`
- `vagrant ssh`
- `cd /vagrant`
- `yarn install`

Reply:
Updated the list and re-ordered.